index.js

```diff
@@ -29,189 +30,73 @@ var api = new ParseServer({
-            });
-        });
-
-        app.get('/yesterday', function(req, res) {
-          var post=null;
-          var aPost = Parse.Object.extend("Posts");
-          var query = new Parse.Query(aPost);
-
-          isLoggedIn(req).then(function(loggedIn){
-          query.descending("from_length");
-          query.equalTo("voted",true);
-          query.limit(10);
-          query.greaterThan('createdAt',new Date(new Date()-2*24*3600000));
-          query.lessThan('createdAt',new Date(new Date()-24*3600000));
-                 query.find({
-                  success: function(posts) {
-                    if(posts!==undefined&&posts.length!==0)
-                    {
-                        res.render('main.ejs', {bot:process.env.BOT,posts: posts,active:2,loggedIn:loggedIn,type:0});
-                    }
-                    else
-                    {
-                      console.log('Nothing to show');
-                      res.render('main.ejs', {bot:process.env.BOT,posts: [],active:2,loggedIn:loggedIn,type:0});
-                    }
-                  },error:function(error){console.log(error);}
-                });
-              });
-          });
-
-          app.get('/alltime', function(req, res) {
-            var post=null;
-            var aPost = Parse.Object.extend("Posts");
-            var query = new Parse.Query(aPost);
-
-            isLoggedIn(req).then(function(loggedIn){
-            query.descending("from_length");
-            query.equalTo("voted",true);
-            query.limit(10);
-                   query.find({
-                    success: function(posts) {
-                      if(posts!==undefined&&posts.length!==0)
-                      {
-                          res.render('main.ejs', {bot:process.env.BOT,posts: posts,active:3,loggedIn:loggedIn,type:0});
-                      }
-                      else
-                      {
-                        console.log('Nothing to show');
-                        res.render('main.ejs', {bot:process.env.BOT,posts: [],active:3,loggedIn:loggedIn,type:0});
-                      }
-                    },error:function(error){console.log(error);}
-                  });
-                });
-            });
+//TODO: handle creation new community
+app.get('/createCommunity', function(req, res) {
+  console.log(req.params.community);
+});
 
-            app.get('/login', function(req, res) {
-              if (!req.query.access_token) {
-                      let uri = steem.getLoginURL();
-                      console.log(uri);
-                      res.redirect(uri);
-                  } else {
-                        res.cookie( 'access_token', req.query.access_token, {expire : new Date() + 24*7*3600*1000});
-                        res.redirect("/now");
-                  }
-            });
+//Login via Steemconnect
+app.get('/login', function(req, res) {
+  if (!req.query.access_token) {
+          let uri = steem.getLoginURL();
```

Comment:
`const` > `let`. There is no need to have it as `let` here.

And the same here, this function is mixing different kind of intendation levels like hell. stick to one schema and use it. Maybe you should think about adding some formatters to your IDE or using code formatting tools like [Prettier](https://github.com/prettier/prettier) which do the job for you. JavaScript for example uses a intendation level of 2.